plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/utils/DateTimeUtils.java

```diff
@@ -62,6 +63,10 @@ public static String formatSince(String value, TimeZone timezone) {
 		return value;
 	}
 	public static Date parse(String value) throws ParseException {
+		if(StringUtils.isEmpty(value)) {
+			//prevent null pointer and index out of bounds exceptions.
+			throw new ParseException("Unparseable date: \"" + value + "\"", 0);
```

Comment:
We should just explicitly say the date was not provided.  If its empty, there is nothing significant to add in the exception